### DISCUSSION TITLE
Questions on generating the input format

### DISCUSSION THREAD

**Discussion by jjj1231978** [sgratzl/chartjs-chart-graph#38] [Q&A]
It's a great library. My understanding is that the input format needs to be in a format like this:

[
  { "name": "1" },
  { "name": "11", "parent": 0 },
  { "name": "111", "parent": 1 },
  { "name": "1111", "parent": 2 },
  { "name": "1112", "parent": 2 },
  { "name": "112", "parent": 1 },
  { "name": "1121", "parent": 5 },
  { "name": "1122", "parent": 5 },
  { "name": "113", "parent": 1 },
  { "name": "1131", "parent": 8 },
  { "name": "1132", "parent": 8 },
  { "name": "12", "parent": 0 },
  { "name": "121", "parent": 11 },
  { "name": "1211", "parent": 12 },
  { "name": "1212", "parent": 12 },
  { "name": "122", "parent": 11 },
  { "name": "1221", "parent": 15 },
  { "name": "1222", "parent": 15 },
  { "name": "123", "parent": 11 },
  { "name": "1231", "parent": 18 },
  { "name": "1232", "parent": 18 },
  { "name": "13", "parent": 0 },
  { "name": "131", "parent": 21 }
]

but the problem is that it is not automatically generated by existing packages like scipy.hierarchy, do you know how to transform scipy's output into this kind of format? Thanks so much.

**Comment by sgratzl** [comment]
isn't this more of a question for scipy.hiearchy folks than this chart.js plugin?

**Reply by jjj1231978** [reply]
True. Could be more of an issue on the scipy.hiearchy's side. Just hoping that there is something that can bridge between scipy.hierarchy and this nice library. Please let us know if you see something useful.
